nova/api/openstack/compute/routes.py

```diff
@@ -860,6 +863,7 @@ class APIRouterV21(base_wsgi.Router):
     and method. The URL mapping based on the plain list `ROUTE_LIST` is built
     at here.
     """
+
```

Comment:
Please remove this additional newline so we keep the diff to upstream as small as possible.

Reply:
That line gets added by autopep8 in the pre-commit hook. So, I would have to do that with the checks disabled.

Reply:
https://review.opendev.org/c/openstack/nova/+/897057/1/doc/source/contributor/development-environment.rst

Just recently, there was a proposed update to the docs upstream, stating again that the pre-commit hooks are optional and how to disable them for a commit :D

Reply:
Sure, they are optional, but the point is, it is added because it is required by the default code guidelines as encoded in the tooling. In my mind, the point of keeping the diffs low is to minimize the effort of re-basing our work. White-space changes like these are not creating conflicts.

Reply:
Not a hard requirement from my side, so we might as well keep it.

Still, in my understanding, white space changes need matching context lines. If upstream rewrites the comment and/or adds typing to the parameters of the `__init__()` we could still see a non-applying patch.